### PR TITLE
fix: Inconsistent key convention for each trigger managers

### PR DESCRIPTION
**Pull request by Sayakie**
As you can find in `file changed` tab, I commented out "isSync" variable so we can reuse it again as we need as in the future. Other changes are just correct to "Sync".

- Closes dareharu/TriggerReactor#3

### CODE REVIEW

**Comment by wysohn** [comment]
Thanks!

Was it "Sync" in previous versions as well? Can't really recall

**Comment by Sayakie** [comment]
Oops, that's my fault. When I reviewed them again, I found that the conventions of all kinds of managers were inconsistent.

The codes below list is the keys which is used by trigger managers. 

### 0️⃣ TriggerInfo
https://github.com/TriggerReactor/TriggerReactor/blob/5da0948183dd5981e98626613926417001b3b24d/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/TriggerInfo.java#L13

### 1️⃣ AbstractAreaTriggerManager
https://github.com/TriggerReactor/TriggerReactor/blob/5da0948183dd5981e98626613926417001b3b24d/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/area/AbstractAreaTriggerManager.java#L42-L44

### 2️⃣ AbstractCommandTriggerManager
https://github.com/TriggerReactor/TriggerReactor/blob/5da0948183dd5981e98626613926417001b3b24d/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/command/AbstractCommandTriggerManager.java#L38-L44

### 3️⃣ AbstractCustomTriggerManager
https://github.com/TriggerReactor/TriggerReactor/blob/5da0948183dd5981e98626613926417001b3b24d/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/custom/AbstractCustomTriggerManager.java#L34-L35

### 4️⃣ AbstractInventoryTriggerManager
https://github.com/TriggerReactor/TriggerReactor/blob/5da0948183dd5981e98626613926417001b3b24d/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/inventory/AbstractInventoryTriggerManager.java#L41-L43

### 5️⃣ AbstractRepeatingTriggerManager
https://github.com/TriggerReactor/TriggerReactor/blob/5da0948183dd5981e98626613926417001b3b24d/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/repeating/AbstractRepeatingTriggerManager.java#L40-L41

First, I will revert the above commits and create new ones. For better tracking will proceed with this PR.

We must define a convention for keys that would be used by them. How about enforce with PascalCase? and then should be unify them into `public` or `protected`.

**Comment by wysohn** [comment]
Yeah I blame my past self

Thank you for working on these!

File: core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/command/AbstractCommandTriggerManager.java [hunk 1]
```diff
@@ -35,14 +35,6 @@
 
 
 public abstract class AbstractCommandTriggerManager extends AbstractTriggerManager<CommandTrigger> {
-    private static final String SYNC = "sync";
-    private static final String PERMISSION = "permissions";
-    private static final String ALIASES = "aliases";
-    public static final String TABS = "tabs";
-
-    public static final String HINT = "hint";
-    public static final String CANDIDATES = "candidates";
```
Comment (Sayakie) [inline core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/command/AbstractCommandTriggerManager.java:44]: We should offer a solution to fix inconsistent keys without users having to do do anything else.

File: core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/TriggerInfo.java [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 import io.github.wysohn.triggerreactor.core.config.source.IConfigSource;
+import io.github.wysohn.triggerreactor.core.manager.Manager;
 import io.github.wysohn.triggerreactor.tools.ValidationUtil;
 
 import java.io.File;
 import java.util.Objects;
 import java.util.Optional;
 
 public abstract class TriggerInfo implements IMigratable {
-    public static final String KEY_SYNC = "sync";
```
Comment (Sayakie) [inline core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/TriggerInfo.java:13]: Ditto

**Review by Sayakie — COMMENTED** [review]
I've finished collecting keys into [one place](https://github.com/Sayakie/TriggerReactor/blob/f82128d3aac446e6d28ec0316d679888f05fd444/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/Manager.java#L31-L56), but I must do to provide an way to automatically replace inconsistent keys.

@wysohn, do you have any good ideas?

**Comment by wysohn** [comment]
@Sayakie 
Actually, I'm planning to have them in the XXXTrigger classes, so I want to know what you think about that: https://github.com/TriggerReactor/TriggerReactor/blob/refactor_dagger/core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/command/CommandTrigger.java

**Comment by Sayakie** [comment]
That sounds good. But still I wonder that someone conflicts with "sync" key, and should config keys be with PascalCase?

**Comment by Sayakie** [comment]
Big sorry for late reply.

> So if we were to set a convention here, camalCasing might be better

Yes, I also would like to set camelCase is better.
Thus, 'll close this PR as I know that it will be solved together at the `refactor_dagger` branch.

**Pull request closed without merging**